nipype/algorithms/tests/test_compcor.py

```diff
@@ -0,0 +1,92 @@
+# emacs: -*- mode: python; py-indent-offset: 4; indent-tabs-mode: nil -*-
+# vi: set ft=python sts=4 ts=4 sw=4 et:
+import nipype
+from ...testing import assert_equal, assert_true, assert_false, skipif, utils
+from ..compcor import CompCor, TCompCor, ACompCor
+
+import unittest
+import mock
+import nibabel as nb
+import numpy as np
+import os
+
+class TestCompCor(unittest.TestCase):
+    ''' Note: Tests currently do a poor job of testing functionality '''
+
+    filenames = {'functionalnii': 'compcorfunc.nii',
+                 'masknii': 'compcormask.nii',
+                 'components_file': None}
```

Comment:
Does this look right?: https://github.com/nipy/nipype/pull/1599/commits/08bccfacd7885905b0873abb998f486577eaee63

I based it off of nipype/interfaces/fsl/tests/test_model.py

It is probably worth modifying the testrunner itself, someday, to create a new tempdir for every test class that needs it. Good for namespacing, too, which I was having trouble with a few days ago!